x-pack/plugins/maps/public/classes/sources/kibana_regionmap_source/kibana_regionmap_source.ts

```diff
@@ -49,10 +58,12 @@ export class KibanaRegionmapSource extends AbstractVectorSource {
     ];
   }
 
-  async getVectorFileMeta() {
-    const regionList = getKibanaRegionList();
-    const meta = regionList.find((source) => source.name === this._descriptor.name);
-    if (!meta) {
+  async getVectorFileMeta(): Promise<LayerConfig> {
+    const regionList: LayerConfig[] = getKibanaRegionList() as LayerConfig[];
```

Comment:
Instead of casting results of getKibanaRegionList here, can you update the method definition in meta.ts so it property typed everywhere and does not need a cast?